proto/oracle/v1/query.proto

```diff
@@ -69,6 +69,13 @@ service Query {
       returns (QueryRequestPriceResponse) {
     option (google.api.http).post = "/oracle/request_prices";
   }
+
+  // RequestVerification verifies a request to make sure that
+  // all information that will be used to report the data is valid
+  rpc RequestVerification(QueryRequestVerificationRequest)
+    returns (QueryRequestVerificationResponse) {
+      option (google.api.http).post = "/oracle/verify_request";
```

Comment:
please change to '/oracle/v1/verify_request'

Reply:
Fix. Test passed, yay.